langlib/lang.int/src/main/ballerina/int.bal

```diff
@@ -109,6 +130,15 @@ public isolated function max(int n, int... ns) returns int = @java:Method {
 
 # Returns the minimum of one or more int values.
 #
+# ```ballerina
+# // Retrieving the lowest value out of the list of integers
```

Comment:
same as max

Reply:
Comment is added in order to separate the two examples.